dotcom-rendering/src/components/LoopVideoABTest.tsx

```diff
@@ -0,0 +1,27 @@
+import { css } from '@emotion/react';
+
+export const LoopVideoABTest = () => (
+	<video
+		id="video1"
+		autoPlay={true}
+		loop={true}
+		muted={true}
+		playsInline={true}
+		controls={true}
+		preload="none"
```

Comment:
non-blocking:
It looks like `preload`[ is ignored when autoplay is set to true](https://www.w3schools.com/tags/att_video_preload.asp), as it has to load on page load.

It might be best to remove this to reduce confusion about the load.

Reply:
That's a good point. Since this is just a test for CWVs that we'll remove after, I'll leave this for now.